app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -90,9 +94,15 @@ public Retrofit getRetrofit() {
                 .retryOnConnectionFailure(true)
                 .build();
 
+        /**
+         * Gson object used in convert factory for serialization and deserialization of objects
+         */
+        Gson gson = new GsonBuilder().setLenient().create();
+
         if (retrofit == null) {
             retrofit = new Retrofit.Builder()
                     .baseUrl(BuildConfig.REST_URL + "/")
+                    .addConverterFactory(GsonConverterFactory.create(gson))
```

Comment:
Why is it necessary to use a converter beyond the default Retrofit configuration?